<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1122</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>White Rabbit Trigger Demo 2012</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Devices:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TX Cores:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="QGroupBox" name="txGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>TX Core Details:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QSpinBox" name="txTriggerIdSpinbox">
           <property name="maximum">
            <number>4095</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Trigger ID:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="txReinitCoreButton">
           <property name="text">
            <string>Reinitalize core</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>RX Cores:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QListWidget" name="txCoreList">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QListWidget" name="rxCoreList">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1" rowspan="3">
       <widget class="QListWidget" name="deviceList">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="4" rowspan="2">
       <widget class="QGroupBox" name="rxGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>RX Core Details:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Trigger ID:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Triggers executed:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Triggers received:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="rxTriggersExec">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="rxTriggersRcvd">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" rowspan="2">
          <widget class="QPushButton" name="rxResetCountersButton">
           <property name="text">
            <string>Reset Triggers</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Output delay:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="3">
          <widget class="QComboBox" name="rxTriggerIdCombobox"/>
         </item>
         <item row="1" column="4" rowspan="2">
          <widget class="QPushButton" name="rxReinitCoreButton">
           <property name="text">
            <string>Reinitialize Core</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="3">
          <widget class="QDoubleSpinBox" name="rxTriggerDelaySpinBox">
           <property name="suffix">
            <string> ns</string>
           </property>
           <property name="minimum">
            <double>5000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>15000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="4">
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="histogramPainter" name="rxHistogramPainter" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>540</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>512</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>histogramPainter</class>
   <extends>QWidget</extends>
   <header location="global">histogrampainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
